Sources/MongoMobile/MongoMobile.swift

```diff
@@ -140,6 +134,17 @@ public class MongoMobile {
         if let cachedInstance = embeddedInstances[settings.dbPath] {
             instance = cachedInstance
         } else {
+            // NOTE: This hack can be removed once SERVER-38943 is resolved. Also note
+            //       that the destroy code is not refactored into a common function because
+            //       we should be removing this code in the near future.
+            if embeddedInstances.count > 0 {
```

Comment:
[nit] !embeddedInstances.isEmpty (again this is according to MongoSwift linting, not sure if it applies here)